lib/kube/proxy/forwarder.go

```diff
@@ -156,7 +171,7 @@ func (f *ForwarderConfig) CheckAndSetDefaults() error {
 	if f.Component == "" {
 		f.Component = "kube_forwarder"
 	}
```

Comment:
Validate that `KubeServiceType` is one of the 3 known values

Reply:
done in latest commit